backend/test.js

```diff
@@ -0,0 +1,150 @@
+require("dotenv").config();
+const { Pool } = require("pg");
+
+// Documentation for pg: https://node-postgres.com/
+//load environment variables
+const config = {
+    user: process.env.DATABASE_USER,
+    host: process.env.DATABASE_HOST,
+    database: process.env.DATABASE_NAME,
+    password: process.env.DATABASE_PASSWORD,
+    port: process.env.DATABASE_PORT
+};
+
+const pool = new Pool(config);
+
+pool.query("SELECT NOW()", (err, res) => {
+    if (err) {
+        console.error(err);
+    } else {
+        console.log("Connection successful");
+    }
+    //pool.end();
+});
+
+////////////////////////////////////////////////////////////////
+
+/**
+ * QUERIES THAT DEAL WITH USERS
+ */
+
+//get users
+const getUsers = async (request, response) => {
+    try {
+        const result = await pool.query("SELECT * FROM users ORDER BY id ASC");
+        response.status(200).json(result.rows);
+    } catch (error) {
+        throw error;
+    }
+};
+// get a single user
+const getUserById = (request, response) => {
+    const id = parseInt(request.params.id);
```

Comment:
Unhandled exception here, when non int param is passed. Perhaps try catch the function instead of using callbacks.

Reply:
changed to example functions